### PR TITLE
Better diff mode (DIFFABLE=1)

### PR DESCRIPTION
**Pull request by wilzbach**
Follow-up to https://github.com/dlang/tools/pull/222 and https://github.com/dlang/dlang.org/pull/1537/

>> Btw as this could also lead to unrelated diffs, should we disable the Bugzilla query for bugs for the changelog builder in "nightly mode"?

> Hmm, yeah, changes that shouldn't result in documentation changes should not have documentation changes indicated in the status line, because then inadvertent documentation changes will get through. I guess with this and https://github.com/dlang/dlang.org/pull/1537 we should expand `NODATETIME` to something more generic.

### CODE REVIEW

File: dpl-docs/views/layout.dt [hunk 1]
```diff
@@ -6,6 +6,7 @@ html(lang='en-US')
     | http://digitalmars.com
   - import std.process : environment;
   - string version_id = environment["GIT_TARGET"];
+  - bool noExactSourceCodeLinks = environment["NO_EXACT_SOURCE_CODE_LINKS"] == "1";
```
Comment (wilzbach) [inline dpl-docs/views/layout.dt]: Apparently that's the "workaround" to inject parameters into the layout file - it's also the prevalent style here (see above).

**Comment by CyberShadow** [comment]
```
[2017-02-27 09:13:16.384055] Running: 'make' '-f' 'posix.mak' 'DIFFABLE=1' 'all' 'kindle' 'pdf' 'verbatim'
```

Seems to be working from my side :)

**Comment by CyberShadow** [comment]
Relevant change: https://github.com/CyberShadow/ae/commit/a3134576583d1919995a85b8282cf955b1cfd7de

**Comment by CyberShadow** [comment]
Why the broken "Previous version.html" link? Will that be resolved by https://github.com/dlang/tools/pull/222?

**Comment by wilzbach** [comment]
> Why the broken "Previous version.html" link? Will that be resolved by dlang/tools#222?

Good catch!
It's due to the `changed.d` trying to detect the previous version based on the git revision pattern, which isn't passed anymore. Luckily there's the `--prev-version` flag (fixed).

**Pull request merged**